.env.example

```
@@ -18,14 +18,12 @@ OUTPUT_DIR=./outputs
+R2_ACCESS_KEY_ID=
+R2_SECRET_ACCESS_KEY=
+R2_BUCKET_NAME=
+R2_ENDPOINT_URL=https://<ACCOUNT_ID>.r2.cloudflarestorage.com
+R2_PUBLIC_URL=https://pub-xxxxx.r2.dev
 
 # marker
-MARKER_USE_GPU=false
-
-# Pandoc
-PANDOC_PDF_ENGINE=xelatex
+MARKER_USE_GPU=false
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)